packages/hebao_v1/contracts/modules/security/GuardianModule.sol

```diff
@@ -70,53 +61,27 @@ contract GuardianModule is SecurityModule
     {
         require(guardian != address(0), "ZERO_ADDRESS");
         require(group < GuardianUtils.MAX_NUM_GROUPS(), "INVALID_GROUP");
+        uint numGuardians = controller.securityStore().numGuardiansWithPending(wallet);
+        require(numGuardians < MAX_GUARDIANS, "TOO_MANY_GUARDIANS");
 
-        if (controller.securityStore().numGuardians(wallet) == 0) {
-            controller.securityStore().addOrUpdateGuardian(wallet, guardian, group);
-            emit GuardianAdded(wallet, guardian, group);
-        } else {
-            uint confirmStart = pendingAdditions[wallet][guardian][group];
-            require(confirmStart == 0 || now > confirmStart + confirmPeriod, "ALREADY_PENDING");
-            pendingAdditions[wallet][guardian][group] = now + pendingPeriod;
-            emit GuardianAdditionPending(wallet, guardian, group, now + pendingPeriod);
+        uint effectiveTime = now;
+        if (controller.securityStore().numGuardians(wallet) > 0) {
```

Comment:
May as well just use `numGuardians` here which will work the same.

Reply:
fixed